src/pages/Refund.tsx

```diff
@@ -19,6 +19,8 @@ const Refund = () => {
     const { updateSwapStatus, wasmSupported, swaps, t } = useGlobalContext();
     const { setTimeoutEta, setTimeoutBlockheight } = usePayContext();
 
+    const [swapFound, setSwapFound] = createSignal(null);
```

Comment:
I don't like having this as signal. We will loose track of what signal does what in which combination